primefaces/src/main/java/org/primefaces/behavior/ajax/AjaxBehaviorListenerImpl.java

```diff
@@ -77,6 +78,10 @@ public void processAjaxBehavior(AjaxBehaviorEvent event) throws AbortProcessingE
         catch (ArrayIndexOutOfBoundsException e) {
             processArgListener(context, elContext, event);
         }
+        // Mojarra hack for Composite Components, see #7819
+        catch (PropertyNotFoundException e) {
+            processArgListener(context, elContext, event);
+        }
```

Comment:
Should this just be a mutli catch `ArrayIndexOutOfBounds | PropertyNotFoundException` with the comment saying `// PropertyNotFoundException is Mojarra hack for Composite Components, see #7819`